packages/perseus/src/components/input-with-examples.jsx

```diff
@@ -14,11 +14,9 @@ import TextInput from "./text-input.jsx";
 import Tooltip from "./tooltip.jsx";
 
 import type {LinterContextProps} from "../types.js";
+import type {StyleType} from "@khanacademy/wonder-blocks-core";
 
 const {captureScratchpadTouchStart} = Util;
-const MATH = "math";
-const TEXT = "text";
-const TEX = "tex";
```

Comment:
Is there a reason we removed this? My first reaction is that it would keep us from misspellings (`"tax"` instead of `"tex"`), but I guess Flow would still catch that since `type` isn't a generic String but a String union.

NAN, the more I think about it the more I like the change.

Reply:
Flow will still catch misspellings as the prop type is `"text" | "tex" | "math"`... so anything other than those three literals will not pass.  I changed this because Flow couldn't understand the `(this.props.type: empty)` assertion (which I added to ensure our `switch case` covered all "enum values".